tests/test_channel.py

```diff
@@ -6,7 +6,7 @@
 import astshim as ast
 from astshim.test import MappingTestCase
 
-DataDir = os.path.join(os.path.dirname(__file__))
+DataDir = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
Is this really the standard approach? I thought one would want to put this in `setUp` or `setUpClass`?

Reply:
Fair enough. I'll change all three tests that use `DataDir` to set `self.dataDir in `setUp`